NGitLab/ITagClient.cs

```diff
@@ -9,10 +9,10 @@ public interface ITagClient
 
         void Delete(string name);
 
-        IEnumerable<Tag> All { get; }
+        ReleaseInfo CreateRelease(string name, ReleaseCreate data);
 
-        RealeaseInfo CreateRelease(string name, ReleaseCreate data);
+        ReleaseInfo UpdateRelease(string name, ReleaseUpdate data);
 
-        RealeaseInfo UpdateRelease(string name, ReleaseUpdate data);
+        IEnumerable<Tag> All { get; }
```

Comment:
The method order was messed up. Any reason?

Reply:
No reason, will correct it in a commit

Reply:
removing the `CreateRelease` and `UpdateRelease` fixed this